cmd/monaco/integration_test_utils.go

```diff
@@ -138,9 +138,15 @@ func cleanupIntegrationTest(t *testing.T, fs afero.Fs, envFile, suffix string) {
 		assert.NilError(t, err)
 
 		for _, api := range apis {
+			if api.GetId() == "calculated-metrics-log" {
+				t.Logf("Skipping cleanup of legacy log monitoring API")
+				continue
+			}
 
 			values, err := client.List(api)
-			assert.NilError(t, err)
+			if err != nil {
+				t.Logf("Failed to cleanup any test configs of type %q: %v", api.GetId(), err)
+			}
 
 			for _, value := range values {
 				// For the calculated-metrics-log API, the suffix is part of the ID, not name
```

Comment:
Since we skip `calculated-metrics-log` now, we could simplify this code now?

Reply:
We could drop the OR case in the case that checks for the suffix in the ID - in the v2 version of this, I've actually done this, but have neglected it here it seems